test/issue-strategies/deployment-task-test.js

```diff
@@ -0,0 +1,10 @@
+import * as notAllowedStrat from '../../src/issue-strategies/not-allowed.js';
+import DummyJira from '../dummy-jira.js';
+
+let dummyJira = new DummyJira();
+
+describe('Deployment Task Strategy Apply Tests', () => {
+  it('Should not be able to commit against Deployment Task, should throw error', () =>
+    notAllowedStrat.apply(dummyJira.issues.DeploymentTask1).should.eventually.be.rejectedWith(Error)
```

Comment:
This test will pass if DeploymentTask1 does not exist within dummy jira, or if any error occurs at all, not just the one you expect.  `rejectedWith` can take a regex parameter that will only pass if the error message matches the regex you supply.  This can be used to ensure you get the error you want.

I'm not sure I should hold up the pull request for this, but it is something to be considered.